docs/synthetics/status-pages.md

```diff
@@ -33,6 +33,14 @@ Every status page will have a unique public URL in this format:
 
 You can use this URL to preview the status page before opening to the public. Disabling the status page will block the public access for the status page.
 
+If you want to **restrict access** to the status page, you can enable **password protection** within the create new status page below Header settings.
```

Comment:
@fulyauluturk I was a little bit confused with this sentence. Is the rephrasing  below correct? 
If you want to **restrict access** to the status page, enable **password protection** in the "Create New Status Page", which is located below the Header settings. 